docs/content/docs/operations/signing-bundles.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Signing Bundles
+description: Signing of Porter bundles
+weight: 7
+---
+
+{{< callout type="info" >}}
+  Signing is supported from v1.1.0
+{{< /callout >}}
+
+Porter has built-in support for signing bundles and the associated invocation image using [Cosign] or [Notation].
```

Comment:
We may need to call out this is in version 1.1.0

Reply:
Agree, are you thinking about something specific?

Reply:
your addition is perfect